### ISSUE TITLE
Annotation file not found error

### ISSUE THREAD

**Issue by abreal555** [webanno/webanno#157]
Hi 
I just installed Webanno on tomcat server 
when I tried to annotate text file this error msg shown 
"Annotation file [h.txt] not found in [/root/.webanno/repository/project/2/document/3/source/h.txt]"

how to fix this problem ?

thanks

**Comment by reckart** [comment]
Was your file really named "h.txt" or did it maybe contain some special character that was somehow truncated?

Does the file exist on the specified path?

**Comment by abreal555** [comment]
yes the file name is h.txt I uploaded it using create project page. I have to mention that I did not use www-data in the deployment. is this going to affect the process of using webanno? also when I checked the server the folder root/.webanno do not contains source folder 

thanks

**Comment by reckart** [comment]
What setup are you using? MySQL yes/no? Standalone or WAR?

**Comment by abreal555** [comment]
I used Mysql  and I deployed the webanno as WAR

**Comment by abreal555** [comment]
more information about how I deployed webanno. I have server with tomcat already installed on with name tomcat so I skipped the steps of installing tomcat from administrator guideline. when I tried to locate properties file I could not find this file settings.properties   so I am working with demo database.  is it because I already have tomcat server with name tomcat not webanno should I have a dedicated server for the webanno?
